.github/stale.yml

```
@@ -0,0 +1,29 @@
+# Probot: Stale
+# A GitHub App built with Probot that closes abandoned Issues and Pull Requests after a period of inactivity.
+# https://github.com/probot/stale
+
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 365
+
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 14
+
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - "critical: roadblock 🚧"
+  - "type: regression 🚨"
+
+# Label to use when marking an issue as stale
+staleLabel: "type: stale 💤"
+
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
```

Reading through it again, I'd remove the last sentence. It sounds almost passive aggressive to me.

```suggestion
  recent activity. It will be closed if no further activity occurs.
```